src/main/java/net/sf/jabref/bibtex/DuplicateCheck.java

```diff
@@ -47,6 +50,9 @@
     // Extra weighting of those fields that are most likely to provide correct duplicate detection:
     private static final HashMap<String, Double> fieldWeights = new HashMap<>();
 
+    private static final Log LOGGER = LogFactory.getLog(DuplicateCheck.class);
```

Comment:
I would tend to add the logger always in the first line after class declaration..